cmd/monaco/download/downloaders.go

```diff
@@ -17,12 +17,38 @@
 package download
 
 import (
+	"context"
+	"github.com/dynatrace/dynatrace-configuration-as-code/cmd/monaco/dynatrace"
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/concurrency"
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/environment"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client/automation"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client/dtclient"
 	v2 "github.com/dynatrace/dynatrace-configuration-as-code/pkg/config/v2"
 	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/download"
+	dlautomation "github.com/dynatrace/dynatrace-configuration-as-code/pkg/download/automation"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/download/classic"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/download/settings"
 )
 
 type downloaders []interface{}
 
+func makeDownloaders(options downloadConfigsOptions) (downloaders, error) {
+	dtClient, err := dynatrace.CreateClient(options.environmentURL, options.auth, false,
+		dtclient.WithClientRequestLimiter(concurrency.NewLimiter(environment.GetEnvValueIntLog(environment.ConcurrentRequestsEnvKey))))
+	if err != nil {
+		return nil, err
+	}
+
+	autClient := automation.NewClient(options.environmentURL, client.NewOAuthClient(context.TODO(), client.OauthCredentials{
```

Comment:
There is no concurrency limiter on the automation client?

Reply:
i will just add it , its easy